src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -600,9 +601,10 @@ protected void setBelowMaxDepthTransactionLimit(int maxAnalyzedTransactions) {
     }
 
     public interface Defaults {
-        //API
+		//API
```

Comment:
fix indentation

Reply:
Not sure why it did that, i checked in code and it doesnt look like that :o

Reply:
It might happen if you use tabs instead of spaces (https://www.youtube.com/watch?v=SsoOG6ZeyUI :trollface: )

Reply:
Hmm I don't see it fixed yet... You can configure your IDE to convert tabs to spaces.